src/client/src/producer.rs

```diff
@@ -25,6 +25,45 @@ impl TopicProducer {
         Self { topic, pool }
     }
 
+    /// Sends a key/value record to this producer's Topic.
+    ///
+    /// The partition that the record will be sent to is derived from the Key.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use fluvio::{TopicProducer, FluvioError};
+    /// # async fn example(producer: &TopicProducer) -> Result<(), FluvioError> {
+    /// producer.send("Key", "Value").await?;
+    /// # Ok(())
+    /// # }
```

Comment:
What's up with this docstring?

Reply:
So this code block will be compiled as Rust code to ensure we don't break the code in the example. In the rust doc-string code-blocks, any line starting with a `#` will not be displayed on the docs page. In this example, the only thing the user will see on docs.rs is 

```
producer.send("Key", "Value").await?;
```

they will not see the rest of this code block, which is nice to reduce visual boilerplate and get right to the meat of the example.

Reply:
Cool.